### PR TITLE
example meta get with optional flags

### PR DESCRIPTION
**Pull request by danmayer**
example adding support for arbitrary meta flags and for getting them back as part of the response

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@ profile.html
 ## Environment normalisation:
 /.bundle/
 /lib/bundler/man/
+dev.yml
```
Comment (oosidat) [inline .gitignore:28]: Leftover from testing?

Comment (danmayer) [reply .gitignore:28]: no, just like other editor files, I intend to add this, you will see it in paquito and other open source libs as well... it makes it a bit easier to work with some of our standard tooling

File: test/integration/test_operations.rb [hunk 2]
```diff
@@ -38,6 +38,27 @@
           end
         end
 
+        it 'return the value and the meta flags' do
+          memcached_persistent(p) do |dc|
```
Comment (oosidat) [inline test/integration/test_operations.rb:42]: Curious: How is `p` set for each test run?

File: lib/dalli/protocol/meta/response_processor.rb [hunk 3]
```diff
@@ -45,6 +45,18 @@ def meta_get_with_value_and_cas
...
         end
 
+        def meta_get_with_value_and_meta_flags(cache_nils: false)
+          tokens = error_on_unexpected!([VA, EN, HD])
+          return [(cache_nils ? ::Dalli::NOT_FOUND : nil), {}] if tokens.first == EN
+
+          meta_flags = meta_flags_from_tokens(tokens)
+          return [(cache_nils ? ::Dalli::NOT_FOUND : nil), meta_flags] unless tokens.first == VA
+
+          value, bitflag = @value_marshaller.retrieve(read_data(tokens[1].to_i), bitflags_from_tokens(tokens))
```
Comment (oosidat) [inline lib/dalli/protocol/meta/response_processor.rb:55]: Can `tokens` be of length 1 (and if yes, do we need to account for that)?

**Pull request merged**